ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/Wizard/ReviewConfig.tsx

```diff
@@ -71,49 +67,40 @@ function ReviewConfig({ clusters, profiles, errorMessage }: ProfileSelectionProp
                 </Flex>
             </PageSection>
             <Divider component="div" />
-            <Stack hasGutter className="pf-v5-u-py-lg pf-v5-u-px-lg">
-                <StackItem>
-                    <Text component={TextVariants.h3} className="pf-v5-u-font-weight-bold">
-                        Name
-                    </Text>
-                    <Text>{formikValues.parameters.name}</Text>
-                </StackItem>
-                <StackItem>
-                    <Text component={TextVariants.h3} className="pf-v5-u-font-weight-bold">
-                        Schedule
-                    </Text>
-                    <Text>{formattedScanSchedule}</Text>
-                </StackItem>
-                <StackItem>
-                    <Text component={TextVariants.h3} className="pf-v5-u-font-weight-bold">
-                        Clusters
-                    </Text>
-                    <TextList isPlain>
+            <Flex
+                direction={{ default: 'column' }}
+                spaceItems={{ default: 'spaceItemsLg' }}
+                className="pf-v5-u-pt-lg pf-v5-u-px-lg"
+            >
+                <ScanConfigParametersView
+                    headingLevel={headingLevel}
+                    scanName={formikValues.parameters.name}
+                    description={formikValues.parameters.description}
+                    scanSchedule={scanSchedule}
+                />
+                <Flex direction={{ default: 'column' }}>
+                    <Flex spaceItems={{ default: 'spaceItemsSm' }}>
+                        <Title headingLevel={headingLevel}>Clusters</Title>
+                        <Badge isRead>{selectedClusters.length}</Badge>
+                    </Flex>
+                    <List isPlain>
                         {selectedClusters.map((cluster) => (
```

Comment:
Outside of the scope of this PR since it was done in last release, but are you concerned about this part for users with a large amount of clusters?